CHANGELOG.md

```diff
@@ -1,41 +1,64 @@
-CHANGELOG
```

Comment:
I would recommend you not to change everything as the existing project's layout was pretty the same as the other projects in the Geocoder org. Yet, feel free to make the changes you wish, but you're not alone ;-)